vm-mock/vm.js

```diff
@@ -243,18 +241,20 @@ export default function createMockedABI(
       },
 
       assembly_script_get_data(kPtr) {
-        let v = '';
         const k = ptrToUint8ArrayString(kPtr);
 
         if (ledger.has(contractAddress)) {
           const addressStorage = ledger.get(contractAddress).storage;
 
           if (addressStorage.has(k)) {
-            v = addressStorage.get(k);
+            return addressStorage.get(k);
```

Comment:
Here you don't use newArrayBuffer. It doesn't seem to affect the tests. Are you sure we don't need it ?